### PR TITLE
PRO-784: Document 'address_32_from_hex' function

### PR DESCRIPTION
**Pull request by helanto**
Documentation for `address_32_from_hex`.

A screenshot of how this thing looks like 👇 

Screenshot 2024-02-13 at 10 59 44 AM

### CODE REVIEW

File: docs/query/DuneSQL-reference/Functions-and-operators/varchar-utility-functions.md [hunk 1]
```diff
@@ -33,4 +33,23 @@ This function returns a string with the first letter of each word in uppercase a
...
+
+#### address_32_from_hex()
+**``address_32_from_hex(varchar)``** → varbinary
+
+This function converts a hexadecimal-formatted input string to varbinary, always producing an output of 32 bytes in length by left-padding with the 0x00 byte. The input string needs to be hexadecimal formatted, allowing characters from `[a-fA-F0-9]`; otherwise, an error is returned. While the input can be prefixed with '0x', this is not mandatory. The function accommodates the hexadecimal representation of any decimal number, provided it fits within 32 bytes.
+
+This function proves useful in scenarios where addresses in the Aptos chain are utilized. In the Aptos chain, addresses serve as 32-byte identifiers for accounts or objects. All addresses in Aptos chain are represented as just that, using a 32-byte varbinary. However, it's quite common to represent addresses in their short-formed representation, having leading zeros trimmed. For example aptos account in address 0x1. Additionally, it's typical to encounter the short format without the '0x' prefix. Therefore, this function serves to transform a short-form address into the internal representation format used for addresses
```
Comment (belen-pruvost) [inline docs/query/DuneSQL-reference/Functions-and-operators/varchar-utility-functions.md:43]: `For example aptos account in address 0x1.`
Could this be rephrased? Something like `For examples, a valid aptos account can have a 0x1 address`

**Review by belen-pruvost — APPROVED** [review]
This looks great! You may want to check with @0xBoxer about whether to merge this directly or not, I think there was some rework being done to the orchestration of docs.

**Comment by belen-pruvost** [comment]
> This looks great! You may want to check with @0xBoxer about whether to merge this directly or not, I think there was some rework being done to the orchestration of docs.

That's probably the reason why the checks here are failing

File: docs/query/DuneSQL-reference/Functions-and-operators/varchar-utility-functions.md (hunk 1 above)
Comment (helanto) [reply docs/query/DuneSQL-reference/Functions-and-operators/varchar-utility-functions.md]: No clue why they are failing. I don't have access to vercel to check. However, I notice that all vercel checks in this repo are failing recently. Maybe @0xBoxer knows more.

Comment (belen-pruvost) [reply docs/query/DuneSQL-reference/Functions-and-operators/varchar-utility-functions.md]: Yes - it's unrelated to your changes 🙏

**Review by hawkaa — APPROVED** [review]
LGTM!

**Comment by helanto** [comment]
This is a duplicate from [dune-api-docs#40](https://github.com/duneanalytics/dune-api-docs/pull/40). Closing ❌

**Pull request closed without merging**